src/main/java/org/influxdb/dto/Point.java

```diff
@@ -263,10 +264,11 @@ private void addFieldByAttribute(final Object pojo, final Field field, final Col
 
         TimeColumn tc = field.getAnnotation(TimeColumn.class);
         if (tc != null && Instant.class.isAssignableFrom(field.getType())) {
```

Comment:
what should I do if some filed has TimeColumn annotation but filed type not Instant?